mari/scan.c

```diff
@@ -138,10 +138,20 @@ bool mr_scan_select(mr_channel_info_t *best_channel_info, uint32_t ts_scan_start
 
 inline void _save_rssi(size_t idx, mr_beacon_packet_header_t beacon, int8_t rssi, uint8_t channel, uint32_t ts_scan, uint64_t asn_scan) {
     size_t channel_idx                                          = channel % MARI_N_BLE_REGULAR_CHANNELS;
-    scan_vars.scans[idx].channel_info[channel_idx].rssi         = rssi;
-    scan_vars.scans[idx].channel_info[channel_idx].timestamp    = ts_scan;
+    mr_beacon_scan_header_t scan_beacon = {
```

Comment:
Can you add a comment explaining that you are copying all but the bloom filter, so you don't waste memory since the scan module doesn't use the bloom filter.